src/main/resources/config/application-dev.yml

```diff
@@ -35,7 +35,7 @@ spring:
     type: com.zaxxer.hikari.HikariDataSource
     url: jdbc:mysql://localhost:3306/solr?allowPublicKeyRetrieval=true&useUnicode=true&characterEncoding=utf8&useSSL=false&useLegacyDatetimeCode=false&serverTimezone=UTC&createDatabaseIfNotExist=true
     username: root
-    password: admin
```

Comment:
Always try retaining original/master credentials of the root project, and then push these files(in case any addition like _new dependencies_ are done, otherwise ignore staging them only).